xbmc/filesystem/File.cpp

```diff
@@ -66,7 +66,7 @@ bool CFile::Copy(const CURL& url2, const CURL& dest, XFILE::IFileCallback* pCall
   CURL url(url2);
   if (StringUtils::StartsWith(url.Get(), "zip://") || URIUtils::IsInAPK(url.Get()))
     url.SetOptions("?cache=no");
-  if (file.Open(url.Get(), READ_TRUNCATED | READ_CHUNKED))
+  if (file.Open(url.Get(), READ_TRUNCATED | READ_NO_CACHE | READ_NO_BUFFER))
```

Comment:
Same: why is this changed to no-cache?

Reply:
Seems this method is only used to handle Zip files and in any case is not related to playback of audio/video files or Video Player in general. Then this makes sure is not used FileCache or StremBuffer to generic copy files.

Reply:
I guess the same thing applies here as for .iso-files: We need to consider potential small backward/forward seeks.

Reply:
Removed READ_NO_CACHE flag then. Anyway seems FileCache is not used here but I don't know in what cases this code is really used: `CFile::Copy`

Reply:
Seems self contained method that uses it's own buffer to copy files:

https://github.com/xbmc/xbmc/blob/426d10df804198c3402f231f72bbdf98029d61d9/xbmc/filesystem/File.cpp#L119-L187

Reply:
Ah yes, that would mean that caching doesn't do much here.